### PR TITLE
SiteQueryService Refator

### PR DESCRIPTION
**Pull request by IanDTrapp**

### CODE REVIEW

File: include/service/entities/site_query_service.js [hunk 1]
```diff
@@ -28,13 +28,18 @@ module.exports = function SiteQueryServiceModule(pb) {
   /**
    * Create an instance of the site query service specific to the given site
    *
-   * @param {String} siteUId UID of site, should already be sanitized by SiteService
-   * @param {Boolean} onlyThisSite  for q, return results specific to this site instead of also looking in global
+   * @module Services
+   * @class SiteQueryService
    * @constructor
+   * @param {Object} options
+   * @param {String} [options.site=GLOBAL_SITE]
```
Comment (andparker) [inline include/service/entities/site_query_service.js]: Bring back the parameter descriptions!!

File: include/service/entities/site_query_service.js [hunk 2]
```diff
@@ -28,13 +28,18 @@ module.exports = function SiteQueryServiceModule(pb) {
...
+   * @param {String} [options.onlyThisSite=false]
```
Comment (andparker) [inline include/service/entities/site_query_service.js]: Same here!!

**Comment by andparker** [comment]
Hey Ian. Merge 0.5.0 and resolve the conflicts.

**Comment by IanDTrapp** [comment]
Merged and fixed

File: plugins/portfolio/controllers/index.js [hunk 3]
```diff
@@ -38,8 +38,7 @@ module.exports = function IndexModule(pb) {
     Index.prototype.init = function (props, cb) {
         var self = this;
         pb.BaseController.prototype.init.call(self, props, function () {
-            self.siteQueryService = new pb.SiteQueryService(self.site);
-            cb();
```
Comment (wesanderson303) [inline plugins/portfolio/controllers/index.js]: Did you mean to remove this callback?

**Pull request merged**